llvm/tools/repo2obj/repo2obj.cpp

```diff
@@ -99,19 +97,14 @@ void SpecialNames::initialize(const pstore::database &Db, GeneratedNames &Names)
 
 // findString
 // ~~~~~~~~~~
-pstore::typed_address<pstore::indirect_string>
-SpecialNames::findString(pstore::database const &Db,
-                         pstore::index::name_index const &NameIndex,
-                         pstore::indirect_string const &Str) {
-
-  auto Pos = NameIndex.find(Db, Str);
-  return (Pos != NameIndex.end(Db))
-             ? pstore::typed_address<pstore::indirect_string>(Pos.get_address())
-             : pstore::typed_address<pstore::indirect_string>::null();
+auto SpecialNames::findString(pstore::database const &Db,
+                              pstore::index::name_index const &NameIndex,
+                              pstore::indirect_string const &Str) -> istring {
+  auto const Pos = NameIndex.find(Db, Str);
+  return (Pos != NameIndex.end(Db)) ? istring{Pos.get_address()}
+                                    : istring::null();
 }
 
```

Comment:
Again, Is the code refactoring only here? If yes, could you please put into the different commit.

Reply:
As above.

Reply:
Thanks for removing this from this commit. Please feel free to submit your change as a separate commit.